projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts

```diff
@@ -0,0 +1,152 @@
+import { clickHamburger } from '../checkout-flow';
+import { loginRegisteredUser as login } from "../cart";
+
+const HTTP_STATUS_OK = 200;
+const FIRST_ROW = 1;
+const SUBJECT_COLUMN = 1;
+const CATEGORY_COLUMN = 2;
+const STATUS_COLUMN = 5;
+
+const CUSTOMER_SUPPORT_MENU_OPTION_INDEX = 14;
+const FIRST_TICKET_COLUMN_INDEX = 1;
+const FIRST_TICKET_ROW_INDEX = 1;
+
+export enum TestCategory {
+  enquiry = "Enquiry",
+  complaint = "Complaint",
+  problem = "Problem",
+}
+
+export interface TestTicketDetails {
+  subject: string;
+  message: string;
+  category: TestCategory;
+  associatedTo?: string;
+  filename?: string;
+}
+
+export function loginRegisteredUser() {
+  login();
+}
+
+export function clickMyAccountMenuOption(){
+  cy.visit('/');
+  cy.get('#cx-header', { timeout: 10000 }).should('be.visible');
+  cy.onMobile(() => {
+    clickHamburger();
+  });
+  cy.get('.accNavComponent button').click();
+}
+
+export function clickCustomerSupportMenuOption(){
+  cy.get(`.accNavComponent li:nth-child(${CUSTOMER_SUPPORT_MENU_OPTION_INDEX})`).should('contain.text', 'Customer Service');
+  cy.get(`.accNavComponent li:nth-child(${CUSTOMER_SUPPORT_MENU_OPTION_INDEX}) a`).click();
+}
+
+export function verifyTicketListingPageVisit(){
+  cy.url().should('include','/my-account/support-tickets');
+  cy.get('cx-customer-ticketing-list').should('exist');
+}
+
+export function clickFirstTicketFromTicketListing(){
+  cy.get(`#ticketing-list-table tbody tr:nth-child(${FIRST_TICKET_ROW_INDEX}) .cx-ticketing-list-data:nth-child(${FIRST_TICKET_COLUMN_INDEX}) a.cx-ticketing-list-value`).should('exist');
+  cy.get(`#ticketing-list-table tbody tr:nth-child(${FIRST_TICKET_ROW_INDEX}) .cx-ticketing-list-data:nth-child(${FIRST_TICKET_COLUMN_INDEX}) a.cx-ticketing-list-value`).click();
+}
+
+export function verifyTicketDetailsPageVisit(){
+  cy.url().should('match',/http:\/\/.+\/my\-account\/support\-ticket\/[0-9]+/);
+  cy.get('cx-customer-ticketing-messages').should('exist');
+}
+
+export function visitPage(page: string, alias: string){
+  cy.intercept(page).as(alias);
+  cy.visit(page);
+  cy.wait(`@${alias}`).its('response.statusCode').should('eq', HTTP_STATUS_OK);
+}
```

Comment:
Why do we need to parametrize the alias? Can we do:
const alias = `Alias${Date.now()}`;